Dockerfile

```diff
@@ -31,17 +44,15 @@ RUN \
     zip \
     pkg-config \
     upx-ucl \
-    cmake=3.22.1-0kitware1ubuntu20.04.1 \
+    cmake=3.22.2-0kitware1ubuntu20.04.1 \
```

Comment:
We actually can't install an old version this way... it fails to build an image when a new version is out. Bumped to the new version for now.